content/2022/en/LCNC-SEC-10-Security-Logging-and-Monitoring-Failures.md

```diff
@@ -11,6 +11,8 @@ title: "LCNC-SEC-10: Security Logging and Monitoring Failures"
 | --- | --- | --- | --- |
 | 2 | 2 | 3 | 2 |
 
+When an application runs, it's critical to know what ran and the actions it took (also known as an audit trail). This is often done using logging of specific actions. This risk occurs when running these systems log too much or too little information.  
```

Comment:
Maybe we could also mention that in case of a breach, this audit log would be key to understand what happened and how to recover

Reply:
Thank you! Rewrite: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/pull/56/commits/6f11958a6e2ae88b13237bad406adeb8a1df5d12